Tests/Functional/Form/AbstractOverwriteElementTest.php

```diff
@@ -0,0 +1,93 @@
+<?php
+declare(strict_types=1);
+namespace Aoe\AoeDbSequenzer\Tests\Unit\Form;
+
+/***************************************************************
+ *  Copyright notice
+ *
+ *  (c) 2019 AOE GmbH <dev@aoe.com>
+ *
+ *  All rights reserved
+ *
+ *  This script is part of the TYPO3 project. The TYPO3 project is
+ *  free software; you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation; either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  The GNU General Public License can be found at
+ *  http://www.gnu.org/copyleft/gpl.html.
+ *
+ *  This script is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  This copyright notice MUST APPEAR in all copies of the script!
+ ***************************************************************/
+
+use Aoe\AoeDbSequenzer\Domain\Model\OverwriteProtection;
+use Aoe\AoeDbSequenzer\Form\AbstractOverwriteElement;
+use Nimut\TestingFramework\TestCase\FunctionalTestCase;
+
+class AbstractOverwriteElementTest extends FunctionalTestCase
```

Comment:
macht es hier eher vielleicht ein Unit-test Sinn als ein Functional? es ist ja eine "abstract" class.
Die functional tests kann ich mir für die Klassen "OverwriteModeElement" && "OverwriteTillElement" vorstellen.

Reply:
There is only custom rendering methods, so the abstract testing will stay as is.